wallet/wallet.go

```diff
@@ -1259,6 +1261,7 @@ out:
 type txCreateOptions struct {
 	changeKeyScope *waddrmgr.KeyScope
 	selectUtxos    []wire.OutPoint
+	allowUtxo      func(wtxmgr.Credit, int32) bool
```

Comment:
I still haven't been able to convince myself why this needs to take block height as an argument.